studio/components/interfaces/BillingV2/Subscription/Tier/Tier.constants.ts

```diff
@@ -71,7 +73,7 @@ export const SUBSCRIPTION_PLANS: PricingInformation[] = [
     new: false,
     name: 'Enterprise',
     preface: 'For large-scale applications managing serious workloads',
-    features: [`Designated Support manager & SLAs`, `SSO/ SAML`, `Custom contracts & invoicing`],
+    features: [`Designated Support manager & SLAs`, `SSO/SAML`, `Custom contracts & invoicing`],
```

Comment:
![Screenshot 2023-06-01 at 11 27 11](https://github.com/supabase/supabase/assets/14073399/1ad5e433-b8a8-4ce6-8486-1bac9d091749)

+1 for the change, just gotta adjust it everywhere then